config/roles.yml

```diff
@@ -387,6 +393,8 @@ security_analytics_full_access:
     - 'cluster:admin/opensearch/securityanalytics/logtype/*'
     - 'cluster:admin/opensearch/securityanalytics/mapping/*'
     - 'cluster:admin/opensearch/securityanalytics/rule/*'
+    - 'cluster:admin/opensearch/securityanalytics/threatintel/*'
+    - 'cluster:admin/opensearch/securityanalytics/connections/test/s3'
```

Comment:
nit: this looks like a different action for `connections`?

Reply:
This API was implemented alongside threat intel to test the S3 connection which is needed for threat intel but we kept the name as connections as it may be used outside the scope of threat intel.

Reply:
connections/*?